GladiusEx.lua

```diff
@@ -748,7 +748,7 @@ function GladiusEx:CHAT_MSG_SYSTEM(event, msg)
     if InCombatLockdown() then
```

Comment:
Why only in combat?

Reply:
because there is already code for detecting if unitexists and hiding if not exists, but it queues (wait to drop combat) the Hide() function if we are in combat.
I checked all the hide() functions and they all queue (wait to drop combat) 
from this i came to the conclusion that setting GeneralSettings->DeadOpacity to 0
and if someone leaves just set them as dead (no queueing involved) since the original set unit to dead function runs regardless of combat or not

Reply:
Default code sets enemy leaver to stealthed
elseif type == "destroyed" or type == "unseen" then
self:UpdateUnitState(unit, true)
Since ARENA_OPPONENT_UPDATE is fired when enemy leaves with type = destroyed
I think that setting enemy leavers state to DEAD is better than stealthed.

Reply:
Also if you get out of combat it handles them i think problem is in arena you dont drop combat very often and i made it to handle it immediately